### ISSUE TITLE
Weird latex printing of the zero vector

### ISSUE THREAD

**Issue by meganly** [sympy/sympy#21461]
I find it misleading that the `latex_form` of the zero vector, `Vector.zero`, is '\mathbf{\hat{0}}' since the hat notation is usually reserved for unit vectors.  Should it be 'mathbf{0}' instead?

**Comment by Davide-sd** [comment]
I also found `\mathbf{\hat{0}}` to be weird. On my hand notes I always used the `\vec{0}` syntax. I think that there should be an option on the latex printer allowing for custom syntax, just like there is an option to set the imaginary unit to `i` or `j` or something else.

**Comment by meganly** [comment]
Agreed, I would love options for the latex printer to format vectors! The way the latex form is hard-coded in the construction of vectors makes it hard to implement this though...

**Comment by Davide-sd** [comment]
There is a quick hack you could use for the moment, though I wouldn't use it in library codes:

```
from sympy.vector import VectorZero, Vector
VectorZero._latex_form = r'\mathbf{\vec{0}}'
display(Vector.zero)
```

Now, every time vector zero is being rendered, it will use the new attribute. Note that this edit must be done at every session. Once you close the kernel, the edit is gone.

**Comment by oscargus** [comment]
Oh, yeah, you were talking about hats. I just saw this in the book "Mathematics in Type" and though about this issue.
![image](https://user-images.githubusercontent.com/8114497/133254358-a2f38394-bfc9-4e8c-8ce3-2dd55941ac0f.png)

If I read your previous comments, it seems like the best thing would be to provide an option to either print as `\vec{0}` or `\mathbf{0}` (or the hat notation, to not break anything), right?

**Comment by meganly** [comment]
> If I read your previous comments, it seems like the best thing would be to provide an option to either print as `\vec{0}` or `\mathbf{0}` (or the hat notation, to not break anything), right?

Correct.

**Comment by oscargus** [comment]
That is, if anything else but `ZeroVector` should be configurable...
